source/clock.rst

```diff
@@ -0,0 +1,143 @@
+
+In the Northern Hemisphere...
+
+1. Place the watch on a level surface, such as the ground, or hold it horizontal in your hand
+2. Point the hour hand at the sun
+3. Find the angle between the hour hand and the twelve o' clock mark on the watch to find the north-south line
+
+In the Southern Hemisphere...
+
+1. Place the watch on a level surface, such as the ground, or hold it horizontal in your hand
+2. Point the hour hand at the sun
+3. Find the angle between the hour hand and the twelve o' clock mark on the watch to find the north-south line
+
+The centre of the angle between the hour hand and twelve o'clock mark is the north-south line. If you don't know which way is north and which south, just remember that no matter where you are, the sun rises in the east and sets in the west. In the northern hemisphere the sun is due south at midday. If your watch is set to daylight saving time bisect the angle between the hour hand and the one o'clock mark instead.
+
+
+Modifying Clock
+--------------------
+Show me the code
+::::::::::::::::
+You can explore the Clock source code `cloning the project <https://github.com/sugarlabs/clock-activity>`_ directly from git.
+
+Translation and User-Interface
+::::::::::::::::::::::::::::::
+The code for the Clock activity is easily localizable to be used in Sugar with different languages and locales.
+To achieve that, we tried to put into gettext dictionnary all elements which could be culturally different, such as the fonts used to display texts in the clock.
+We also used Pango to have a better font management and give the translators the ability to adapt the display of text to their locale.
+
+To read more about translating Clock, do visit the `wiki <https://github.com/sugarlabs/clock-activity/wiki>`_.
+
+Points of Interest
+::::::::::::::::::
+* Developped at 99% on a XO
+* Icons were hand crafted without a SVG-editor
+* Creating a new GTK+ widget and paint its content.
+* Usage of pango markup to offer better flexibility in localization.
+* Usage of signal `notify::active` sent by Sugar to an activity when it comes on the front. In this activity, we stop refreshing the display when we are not active.
+* Full pydoc documentation.
+* Usage of the badly documented `gobject.threads_init()` to have better messages processing.
+
+
+Reporting Problems
+------------------
+There are several known problems with the current version of Clock:
+
+* Whenever the label where the time is displayed becomes wider than the screen width, for instance when you rotate the display, it would be nice to have line wrapping.
+The `Label.set_line_wrap(True)` method does not give the expected result.
+
+If you discover a bug in the program or have a suggestion for an enhancement, please `file an issue <https://github.com/sugarlabs/clock-activity/issues>`_.
+
+
+Credits
+----------
+Clock was written by Pierre Métras <pierre@alterna.tv> under the Public Domain and GPLv3+ License.
```

Comment:
No, Pierre wrote it and released it as public domain.  The GPLv3 applies to other components added by other people since.